datatypes/src/collections/multi_line_string_collection.rs

```diff
@@ -562,27 +564,32 @@ mod tests {
         )
         .unwrap();
 
-        let expected_collection = MultiLineStringCollection::from_slices(
-            &[
-                MultiLineString::new(vec![vec![MARBURG_EPSG_900_913, HAMBURG_EPSG_900_913]])
-                    .unwrap(),
-                MultiLineString::new(vec![
-                    vec![
-                        COLOGNE_EPSG_900_913,
-                        MARBURG_EPSG_900_913,
-                        HAMBURG_EPSG_900_913,
-                    ],
-                    vec![HAMBURG_EPSG_900_913, COLOGNE_EPSG_900_913],
-                ])
-                .unwrap(),
-            ],
-            &[TimeInterval::default(), TimeInterval::default()],
-            &[("A", FeatureData::Int(vec![1, 2]))],
-        )
-        .unwrap();
+        let expected = [
+            MultiLineString::new(vec![vec![MARBURG_EPSG_900_913, HAMBURG_EPSG_900_913]]).unwrap(),
+            MultiLineString::new(vec![
+                vec![
+                    COLOGNE_EPSG_900_913,
+                    MARBURG_EPSG_900_913,
+                    HAMBURG_EPSG_900_913,
+                ],
+                vec![HAMBURG_EPSG_900_913, COLOGNE_EPSG_900_913],
+            ])
+            .unwrap(),
+        ];
 
         let proj_collection = collection.reproject(&projector).unwrap();
 
-        assert_eq!(proj_collection, expected_collection);
+        proj_collection
+            .geometries()
+            .into_iter()
+            .zip(expected.iter())
+            .for_each(|(a, e)| {
+                assert!(approx_eq!(
+                    MultiLineString,
+                    a.into(),
+                    e.clone(),
+                    epsilon = 0.00001
+                ));
```

Comment:
Why do we have to clone `a` for comparison?

Are `a` and `e` for actual and expected?

Reply:
AssertEq is implemented on MultiLineString and expects other to also be a MultiLineString.

Reply:
Can't this work on a reference?

Reply:
929f0fd